src/networks/MaxCut.jl

```diff
@@ -11,26 +11,37 @@ struct MaxCut{CT<:AbstractEinsum,WT<:Union{NoWeight, Vector}} <: GraphProblem
     code::CT
     graph::SimpleGraph{Int}
     weights::WT
+    fix_config
 end
-function MaxCut(g::SimpleGraph; weights=NoWeight(), openvertices=(), optimizer=GreedyMethod(), simplifier=nothing)
+function MaxCut(g::SimpleGraph; weights=NoWeight(), openvertices=(), fix_config=Dict(), optimizer=GreedyMethod(), simplifier=nothing)
     @assert weights isa NoWeight || length(weights) == ne(g)
     rawcode = EinCode([[minmax(e.src,e.dst)...] for e in Graphs.edges(g)], collect(Int, openvertices))  # labels for edge tensors
-    MaxCut(_optimize_code(rawcode, uniformsize(rawcode, 2), optimizer, simplifier), g, weights)
+    size_dict = uniformsize(rawcode, 2)
+    for key in keys(fix_config)
```

Comment:
This part is good!
I would suggestion wrapping it into a function e.g. `uniformsize_fix(code, dim, fixedvertices)`, because then it can be reused in other problem constructors.